.test/config-simple/config.yml

```
@@ -16,9 +16,13 @@ repo: "https://github.com/snakemake-workflows/transriptome-differential-expressi
 ref:
     # species
     species: "Homo sapiens"
-    
+    # Genome FA or FASTA or FNA
+    genome: "./ngs-test-data/ref/genome.chr21.fa"
+    # Annotation GFF or GTF
+    annotation: "./ngs-test-data/ref/annotation.chr21.gtf"    
```

yamllint

[error] trailing spaces

(trailing-spaces)